app-ios/Native/Sources/Feature/About/Resources/Localizable.xcstrings

```diff
@@ -1,19 +1,54 @@
 {
   "sourceLanguage" : "en",
   "strings" : {
-    "Credits" : {
+    "2025.09.12(Thu) - 13(Fri)" : {
+
+    },
```

Comment:
Thank you for adding the localizable resources, but this information is incorrect.
ref: #256